.github/workflows/validate-submodule-pull-request.yml

```
@@ -27,6 +27,11 @@ jobs:
           JIRA_BASE_URL: ${{ secrets.JIRA_BASE_URL }}
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
+      - name: Assign issue to PR author
+        uses: Lombiq/GitHub-Actions/.github/actions/assign-issue-to-pr-author@issue/OSOE-545
```

```suggestion
      - name: Assign issue to PR author
        if: github.event_name == 'pull_request' && github.event.action == 'opened'
        uses: Lombiq/GitHub-Actions/.github/actions/assign-issue-to-pr-author@issue/OSOE-545
```
I think this should be triggered only when opening the PR.